check_manifest.py

```diff
@@ -394,10 +394,14 @@ def detect(cls, location):
         # .git can be a file for submodules
         return os.path.exists(os.path.join(location, cls.metadata_name))
 
+    def _has_submodules(cls):
+        return os.path.exists(".gitmodules")
+
     def get_versioned_files(self):
         """List all files versioned by git in the current directory."""
+        extra_args = ["--recurse-submodules"] if self._has_submodules() else list()
```

Comment:
`list()` looks a bit weird; the traditional way of spelling an empty list in Python is `[]`.

Reply:
Fixed in cc7c176.